test.js

```diff
@@ -129,7 +99,7 @@ test('disabling one header does not disable the other headers', (t) => {
       'x-dns-prefetch-control': 'off',
```

Comment:
If relevant, there are some other headers that are set by default. You can see that [tested in Helmet's source code](https://github.com/helmetjs/helmet/blob/c813f916a1aef82d586e3b411b7df7c6de0046de/test/index.test.ts#L18-L36).

Reply:
Thanks!